base/applications/sndrec32/sndrec32.cpp

```diff
@@ -455,8 +460,8 @@ WndProc(HWND hWnd,
             for (int i = 0; i < 5; ++i)
```

Comment:
```suggestion
            for (int i = 0; i < _countof(buttons); ++i)
```

Reply:
Throws a -Werror=sign-compare

Reply:
`for (unsigned int i = 0; i < _countof(buttons); ++i)` ?